RadarSDK/RadarAPIClient.m

```diff
@@ -251,10 +251,12 @@ - (void)trackWithLocation:(CLLocation *_Nonnull)location
         params[@"floorLevel"] = @(location.floor.level);
     }
     long nowMs = (long)([NSDate date].timeIntervalSince1970 * 1000);
-    if (!foreground) {
-        long timeInMs = (long)(location.timestamp.timeIntervalSince1970 * 1000);
+    long timeInMs = (long)(location.timestamp.timeIntervalSince1970 * 1000);
+    RadarTrackingOptions *options = [Radar getTrackingOptions];
+    if (!foreground || options.replay != RadarTrackingOptionsReplayNone) {
         params[@"updatedAtMsDiff"] = @(nowMs - timeInMs);
```

Comment:
all track requests should have this updatedAtMsDiff, likely = 0 for normal tracks, but it'll keep our server code path consistent. We also don't need to set locationMs in the request as its not used, a bit of wasted bandwidth, we can send it at line:417 if the request fails.

Reply:
we can basically think that all tracks are replays, but some are replayed immediately.

Reply:
We toss in the `locationMs` for updating the `updatedAtMsDiff` in `RadarAPIHelper`.

Reply:
I had initial hesitancy towards making all `/track`s have `updatedAtMsDiff`, but yea, your model that they'd all have it matches mine. Might as well change that.